projects/cn0577/common/cn0577_bd.tcl

```diff
@@ -0,0 +1,86 @@
+# ltc2387
+
+create_bd_port -dir I ref_clk
+create_bd_port -dir O sampling_clk
+create_bd_port -dir I dco_p
+create_bd_port -dir I dco_n
+create_bd_port -dir O cnv
+create_bd_port -dir I da_p
+create_bd_port -dir I da_n
+create_bd_port -dir I db_p
+create_bd_port -dir I db_n
+create_bd_port -dir O clk_gate
+
+# adc peripheral
+
+ad_ip_instance axi_ltc2387 axi_ltc2387
+ad_ip_parameter axi_ltc2387 CONFIG.ADC_RES $adc_resolution
```

Comment:
We can hardcode resolution to 18 bits for this project

Reply:
Done